indico/modules/events/registration/templates/management/regform_modify.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'events/registration/management/_regform_base.html' %}
+
+{% block description %}
+    {% trans -%}
+        You can configure here your registration form
```

Comment:
That description sounds very strange ("here" should be at the end of the sentence).

But do we need a description on this page at all?